src/neurotechdevkit/rendering/_source.py

```diff
@@ -27,6 +34,33 @@
 }
 
 
+@enum.unique
+class SourceType(enum.Enum):
```

Comment:
@d-lucena I swapped out the `source_is_flat` and `is_point_source` flags for an `Enum` that can carry any info needed about the high-level image-rendering style.

Along with the helper classmethod `from_source`, it makes the calling code (in `scenarios/_base.py`) a bit simpler. Let me know what you think!